src/xmpp/xmpp-im/types.cpp

```diff
@@ -2026,6 +2039,7 @@ bool Message::fromStanza(const Stanza &s, bool useTimeZoneOffset, int timeZoneOf
 
 	t = childElementsByTagNameNS(root, "http://jabber.org/protocol/muc#user", "x").item(0).toElement();
 	if(!t.isNull()) {
+		setMUCUser(true);
```

Comment:
Actually it used to be like that but then I saw other functions and decided to change the style.